app/assets/javascripts/application.js

```diff
@@ -24,11 +25,40 @@ $(document).ready(function() {
 function sellItem(id){
     $(id).toggleClass('hide');
     $(id).toggleClass('show');
+    findSellTotal();
+    updateTotal();
+}
+
+function findSellTotal(){
+  var sold_items = $("ul.character_trades").find(".show")
+  items_total = [0]
+  $.each(sold_items, function(index, sold_item){
```

Comment:
Identifier 'sold_items' is not in camel case.<br>Identifier 'sold_item' is not in camel case.